codegen/src/codegen.ts

```diff
@@ -124,7 +123,7 @@ export async function generate({
       ${tables.map((table) => `"${table.name}": Repository<${getTypeName(table.name)}>;`).join('\n')}
     }> {
       constructor(options?: XataClientOptions) {
-        super({ databaseURL: "https://${config.workspaceID}.xata.sh/db/${config.dbName}", ...options}, links);
+        super({ databaseURL: "${databaseURL}", ...options}, links);
```

Comment:
```suggestion
        super({ databaseURL, ...options}, links);
```

Reply:
Actually no, this is the codegen, so we need to inline the ``databaseURL``